### ISSUE TITLE
Received NaN for numeric attribute `value`

### ISSUE THREAD

**Issue by jamland** [mderrick/react-html5video#125]
I use `react-html5video` component w/ React 16 and React-Router 4.
And it always show me this warning even with default params so I do not pass any custom params to component.
```
Warning: Received NaN for numeric attribute `value`. 
If this is expected, cast the value to a string.
```
The video component works well just wondering why it is here. It is just a bit annoying =)

![image](https://user-images.githubusercontent.com/188568/33130453-1d1cbaac-cf9c-11e7-8b8b-bfac18f2f801.png)

**Comment by mderrick** [comment]
This was fixed in version 2.5.1 which version are you using?

**Comment by jamland** [comment]
You are right! I was using 2.5.0
Upgraded it and warning is gone. Thanks for reply =)
